build/dependencies.props

```diff
@@ -1,7 +1,9 @@
 <Project>
 
   <PropertyGroup Label="SDK Versions">
-    <NetFrameworkVersion>net6.0</NetFrameworkVersion>
+    <NetFrameworkVersionLts>net6.0</NetFrameworkVersionLts>
```

Comment:
I would change the naming here from `NetFramework` to `Net`.

Reply:
And honestly I would use all that net6/7 TFMs in place without intermediate properties like NetFrameworkVersion/NetFrameworkVersionLts/etc.